PLAN.md

```diff
@@ -0,0 +1,68 @@
+# Plan: DB Subcommand for Trie Preimage Dump
+
+## Overview
+Create a new DB subcommand that dumps the current trie (account and storage nodes) as preimages and publishes them to a preimage storage interface (e.g., DynamoDB).
```

Comment:
Curious why do this as a command? Do we ever envision to do it outside of the ExEx or in other situations?

Reply:
I think we could build it into the ExEx. This is the trie state at the first block that we can serve state from. Probably makes sense to just move this to the ExEx and run it if there is no state available for the parent block.